runtime/compiler/runtime/JITServerAOTCache.cpp

```diff
@@ -702,6 +720,14 @@ JITServerAOTCache::printStats(FILE *f) const
    );
    }
 
+bool
+JITServerAOTCache::cacheHasSpace() const
+   {
+   // The AOT cache record allocations are used as a stand-in for the total memory used by all AOT caches
+   auto aotTotalRecordAllocations = TR::Compiler->persistentGlobalMemory()->_totalPersistentAllocations[TR_Memory::JITServerAOTCache];
+   auto aotMaxBytes = TR::CompilationInfo::get()->getPersistentInfo()->getJITServerAOTMaxBytes();
```

Comment:
Maybe we should store _JITServerAOTMaxBytes directly into  `JITServerAOTCache` as a static. 
I am also leaning towards making`cacheHasSpace()` a static method to reinforce the idea that this query refers to all the AOT cache, not just to the current cache object.